paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/AddPaymentMethod.kt

```diff
@@ -200,7 +200,7 @@ internal fun FormFieldValues.transformToPaymentSelection(
     context: Context,
     paymentMethod: SupportedPaymentMethod,
     paymentMethodMetadata: PaymentMethodMetadata,
-): PaymentSelection.New {
+): PaymentSelection {
```

Comment:
We don't have any tests for this yet, mind adding a task to add these as a follow up?

Reply:
will do

Reply:
https://jira.corp.stripe.com/browse/MOBILESDK-1987